pipeline_dp/dp_engine.py

```diff
@@ -38,7 +39,7 @@ def _add_report_stage(self, text):
         self._report_generators[-1].add_stage(text)
 
     def aggregate(self, col, params: AggregateParams,
-                  data_extractors: DataExtractors):  # pylint: disable=unused-argument
+                  data_extractors: DataExtractors):
```

Comment:
Please add a test for this function. Since it's not finished and the result will change in future it doesn't make sense yet to check results. But we can check that the computation graph is correct, i.e. that all needed functions are called with correct parameters. e.g. we can override _bound_contributions and check that arguments is correct (and the same with other methods used in this function)